src/components/NavigationLinks/NavigationLinks.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import './NavigationLinks.scss'
+
+const NavigationLinks = (props = []) => {
+  const navLinks = props.navLinks;
+  return (
+    <>
+      {navLinks &&
+        <ul className='navigation-links'>
+          { navLinks.map((item, index) => {
+            return (
+              <li key={index} className={`${item.isLogo ? 'logo' : ''}`}>
```

Comment:
Avoid using index as key here, it's ok if u know for sure that this array is not editable or no change is going to happen to it after render ( static render array )

Refer to use id or something unique.